### ISSUE TITLE
Just wondering why it's so tied to Contact data type

### ISSUE THREAD

**Issue by tati-nacmedia** [DoodleScheduling/android-material-chips#30]
Would it be possible to create something that takes Countries, for example? Also, how do you insert a chip at a certain position? To me, the add() method is requesting more information that it should. To me, it should just sick:

- thumb url,
- name. 

That's it. It would also be nice to have control on the layout of the chip. Oh well, I keep looking. Google should really release an official library for this.

**Comment by burntapples** [comment]
I agree. This library is better than others but I don't want it for contacts

**Comment by root-ansh** [comment]
i am not quite clear with the comments of this section, but I think i am having a similar problem. I want to use this library  with chips which are ***NOT*** emails . like just some names/number  which will become `A chip` on spacebar_press/ enterButton_press . is it possible?
